app/views/facility_reservations/edit_admin.html.haml

```diff
@@ -1,5 +1,5 @@
 = content_for :head_content do
-  = render :partial => 'shared/headers/calendar'
+  = render 'shared/headers/calendar'
```

Comment:
We're cleaning up the hash syntax here. Can we clean up the single v double quotes?